drivers/dac/dac_sam.c

```diff
@@ -20,7 +20,7 @@
 #include <zephyr/drivers/dac.h>
 #include <zephyr/drivers/pinctrl.h>
 #include <zephyr/drivers/clock_control/atmel_sam_pmc.h>
-
+#include <zephyr/kernel.h>
```

Comment:
This looks like a bug that ought to be handled in a separate PR and backported to v3.3-branch?

Reply:
Yeah, you are probably right ;)

Done: #56357

Will rebase after that one is merged.